src/core/lib/debug/trace.h

```diff
@@ -53,7 +53,8 @@ void grpc_tracer_enable_flag(grpc_core::TraceFlag* flag);
 class TraceFlag {
  public:
   TraceFlag(bool default_enabled, const char* name);
```

Comment:
It's also supposed to have a constexpr constructor, right?

Reply:
I am not sure I understand, but this ctor does not seem to be a constexpr one. Could you elaborate why this is needed? Thanks.

Reply:
"Dynamic initialization of nonlocal variables is discouraged, and in general it is forbidden. However, we do permit it if no aspect of the program depends on the sequencing of this initialization with respect to all other initializations. Under those restrictions, the ordering of the initialization does not make an observable difference."

So I guess having a non-constexpr constructor is ok in this case so long as no other static variable uses anything from `TraceFlag`. Is this true?

Reply:
I guess it is true as in the ctor it adds itself to a global list of tracers, for which the order does not matter...
I would like to make the globals pointers rather than relying on the trivially destructible part, but it is going to be a larger change.